packages/files-ui/src/Components/Modules/FileBrowsers/MoveFileModal.tsx

```diff
@@ -135,11 +137,34 @@ const MoveFileModule = ({ filesToMove, modalOpen, onClose, onCancel, mode }: IMo
       .then(onClose)
       .catch(console.error)
       .finally(() => setIsMovingFile(false))
-
   }
 
   const desktop = useMediaQuery("md")
 
+  const folders = useMemo(() =>
+    filesToMove.filter((f) => f.isFolder)
+  , [filesToMove])
+
+  const isSubFolderOfAnySelectedFolder = useMemo(() => {
+    if(!movePath){
+      return false
+    }
+
+    return folders.some((folder) => {
+      const currentPathWithFolder = `${currentPath}/${folder.name}`
```

Comment:
This should also solve the problem of moving into itself that @asnaith reported
Don't forget to add the import of `getPathWithFile`
```suggestion
      const currentPathWithFolder = getPathWithFile(currentPath, folder.name)
```

Reply:
Thanks a lot, it's def. the way to go and solves Andres's problem from what I can see.